Resilio Connect API/JSMonitor/app.js

```diff
@@ -36,3 +36,21 @@ setTimeout(function() {console.log(enumerateAgents());}, 5000);
 
 console.log(findArrayDiff([1, 5, 17, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160], [1, 5, 17, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 165]));
 
+var prevList, updatedList;
+
+function periodicAgentUpdate() {
+    updatedList = enumerateAgents();
+    prevList = updatedList; 
+
+    if (findArrayDiff(updatedList, prevList).length != 0) {
```

Comment:
This won't work because you made update list equal to prev list so there isn't gonna be any difference between this